apis/fluent-en_GB/atrium-api-fluent-en_GB-common/src/test/kotlin/ch/tutteli/atrium/api/fluent/en_GB/CharSequenceContainsRegexAssertionsSpec.kt

```diff
@@ -34,11 +34,11 @@ class CharSequenceContainsRegexAssertionsSpec : Spek({
 
         private fun getAtLeastTripleString() =
             { what: String, times: String -> "$contains $what $atLeast $times" } to
-                ("$contains.$atLeast.$regex" to Companion::containsAtLeastString)
+                ("$contains.$atLeast.$regex" to ::containsAtLeastString)
 
         private fun getAtLeastTripleRegex() =
             { what: String, times: String -> "$contains $what $atLeast $times" } to
-                ("$contains.$atLeast.$regex" to Companion::containsAtLeastRegex)
+                ("$contains.$atLeast.$regex" to ::getAtLeastIgnoringCaseTripleString)
```

Comment:
that's a mistake, keep `containsAtLeastRegex` as before

Reply:
If I delete `getAtLeastIgnoringCaseTripleRegex` and `getAtMostIgnoringCaseTripleRegex`, what do I replace them with in the test's arguments below?
```kotlin
    object RegexSpec : ch.tutteli.atrium.specs.integration.CharSequenceContainsRegexAssertionsSpec(
        getNameContainsRegex(),
        getAtLeastTripleRegex(),
        getAtLeastIgnoringCaseTripleRegex(),
        getShortcutTripleRegex(),
        getAtMostTripleRegex(),
        getAtMostIgnoringCaseTripleRegex(),
        "◆ ", "⚬ "
    )
```

Reply:
My bad, you are right. So keep both :)